src/app/components/cart/cart-list/cart-list.component.ts

```diff
@@ -9,15 +9,45 @@ import { CartService } from '../../../services/cart.service';
   changeDetection: ChangeDetectionStrategy.Default,
 })
 export class CartListComponent implements OnInit {
-  booksInCart: ICart[] = [];
+  cart: ICart = {
+    cartItems: [],
+    totalQty: 0,
+    totalCost: 0,
+  };
+
+  @Output() increaseQty = new EventEmitter<number>();
```

Comment:
If I'm not mistaken these outputs are not used. You can remove them

Reply:
Yes, they are the leftovers from copying from another component.
Deleted.